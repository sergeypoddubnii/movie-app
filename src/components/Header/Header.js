import React from 'react';
import { NavLink } from 'react-router-dom';
import routes from '../../routes';
import './Header.scss';
const Header = () => {
  return (
    <header className="header">
      <nav className="nav">
        <ul className="nav__list">
          <li className="nav__item">
            <NavLink to={routes.HOME_PAGE} className="nav__link">
              home
            </NavLink>
          </li>
          <li className="nav__item">
            <NavLink to={routes.FAVOTIRE_MOVIES_PAGE} className="nav__link">
              favorite
            </NavLink>
          </li>
        </ul>
      </nav>
    </header>
  );
};

export default Header;
