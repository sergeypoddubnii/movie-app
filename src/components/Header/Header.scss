.header {
  background: #414141;
  // padding-bottom: 20px;
  .nav {
    padding: 30px 0 30px 30px;
    .nav__list {
      display: flex;
      .nav__item {
        margin-right: 20px;
        .nav__link {
          text-decoration: none;
          color: #fff;
          font-size: 18px;
          letter-spacing: 2px;
          text-transform: uppercase;
          &:hover {
            transform: scale(1.05);
            text-decoration: underline;
          }
        }
      }
    }
  }
}
