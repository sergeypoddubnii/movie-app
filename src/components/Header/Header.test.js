import { shallow } from 'enzyme';
import React from 'react';
import Header from './Header';

const getComponent = props => shallow(<Header {...props} />);

describe('should render component Header', () => {
  it('should render component Header', () => {
    const component = getComponent();
    expect(component).toMatchSnapshot();
  });
});
