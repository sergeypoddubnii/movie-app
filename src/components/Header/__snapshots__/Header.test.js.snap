// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Header should render component Header 1`] = `
<header
  className="header"
>
  <nav
    className="nav"
  >
    <ul
      className="nav__list"
    >
      <li>
        hi
      </li>
      <li
        className="nav__item"
      >
        <NavLink
          className="nav__link"
          to="/home"
        >
          home
        </NavLink>
      </li>
      <li
        className="nav__item"
      >
        <NavLink
          className="nav__link"
          to="/favoritemovies"
        >
          favorite
        </NavLink>
      </li>
    </ul>
  </nav>
</header>
`;
